src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/HeaderTimeChecksPoARule.cs

```diff
@@ -33,28 +34,21 @@ public override void Run(RuleContext context)
             ChainedHeader chainedHeader = context.ValidationContext.ChainedHeaderToValidate;
 
             // Timestamp should be greater than timestamp of prev block.
-            if (chainedHeader.Header.Time <= chainedHeader.Previous.Header.Time)
+            if (chainedHeader.Header.BlockTime <= chainedHeader.Previous.Header.BlockTime)
             {
                 this.Logger.LogTrace("(-)[TIME_TOO_OLD]");
                 ConsensusErrors.TimeTooOld.Throw();
             }
 
             // Timestamp shouldn't be more than current time plus max future drift.
-            long maxValidTime = this.Parent.DateTimeProvider.GetAdjustedTimeAsUnixTimestamp() + MaxFutureDriftSeconds;
-            if (chainedHeader.Header.Time > maxValidTime)
+            DateTime maxValidTime = this.Parent.DateTimeProvider.GetAdjustedTime() + TimeSpan.FromSeconds(MaxFutureDriftSeconds);
+            if (chainedHeader.Header.BlockTime > maxValidTime)
             {
                 this.Logger.LogWarning("Peer presented header with timestamp that is too far in to the future. Header was ignored." +
                                        " If you see this message a lot consider checking if your computer's time is correct.");
                 this.Logger.LogTrace("(-)[TIME_TOO_NEW]");
                 ConsensusErrors.TimeTooNew.Throw();
             }
-
-            // Timestamp should be divisible by target spacing.
-            if (!this.slotsManager.IsValidTimestamp(chainedHeader.Header.Time))
```

Comment:
Why is this not required any longer?

Reply:
We will be focusing more on "round time" vs "target spacing". I.e. "target" spacing will be calculated from round time and i don't think this restriction is particularly helpful.